examples/multi_databases.py

```diff
@@ -0,0 +1,100 @@
+from sqlalchemy import BigInteger, Column, String
+from tornado.gen import coroutine
+from tornado.ioloop import IOLoop
+from tornado.web import Application, RequestHandler
+
+from tornado_sqlalchemy import (
+    SessionMixin,
+    as_future,
+    set_max_workers,
+    SQLAlchemy
+)
+
+
+db = SQLAlchemy()
+
+set_max_workers(10)
+
+
+class User(db.Model):
+    __tablename__ = 'users'
+
+    id = Column(BigInteger, primary_key=True)
+    username = Column(String(255))
+
+
+class Foo(db.Model):
+    __bind_key__ = 'foo'
+    __tablename__ = 'foo'
+
+    id = Column(BigInteger, primary_key=True)
+    foo = Column(String(255))
+
+
+class SynchronousRequestHandler(SessionMixin, RequestHandler):
+
+    def get(self):
+        with self.make_session() as session:
+            count = session.query(User).count()
+
+        # OR count = self.session.query(User).count()
+
+        self.write('{} users so far!'.format(count))
+
+
+class GenCoroutinesRequestHandler(SessionMixin, RequestHandler):
+    @coroutine
+    def get(self):
+        with self.make_session() as session:
+            session.add(User(username='b'))
+            session.add(Foo(foo='foo'))
+            count = yield as_future(session.query(User).count)
+
+        self.write('{} users so far!'.format(count))
+
+
+class NativeCoroutinesRequestHandler(SessionMixin, RequestHandler):
+    async def get(self):
+        with self.make_session() as session:
+            print(session)
+            session.add(User(username='c'))
+            session.add(Foo(foo='d'))
+            count = await as_future(session.query(User).count)
+
+        self.write('{} users so far!'.format(count))
+
+
+if __name__ == '__main__':
+
+    app = Application(
+        [
+            (r'/sync', SynchronousRequestHandler),
+            (r'/gen-coroutines', GenCoroutinesRequestHandler),
+            (r'/native-coroutines', NativeCoroutinesRequestHandler),
+        ],
+        sqlalchemy_database_uri='mysql://t_sa:t_sa@localhost/t_sa',
```

Comment:
I'm thinking if it makes more sense to provide only a `db` instance to the `Application` constructor. This would then avoid the need for the `init_app` function.

Also, the `Application` basically needs a configured database instance, so I don't know if there's any point passing it the configuration options instead.

Reply:
Yes, If we move sqlalchemy_database_uri and sqlalchemy_database_binds to SQLAlchemy,  it will be more clear

Reply:
I learned too much from flask，: -( , flask is used to put all settings in app.settings

Reply:
> @dingyaguang117 also, I've just enabled a `black` lint check in Travis, so that the code style remains consistent now that someone else apart from me is helping out with this project. :) could you pull in the latest changes too?

Good idea, let me do it

Reply:
80. But this is already configured in `pyproject.toml`. So if you run `black` in the project directory, it should already see that setting.